main/src/org/destinationsol/assets/Assets.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.assets;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import org.destinationsol.assets.atlas.Atlas;
+import org.destinationsol.assets.audio.OggMusic;
+import org.destinationsol.assets.audio.OggSound;
+import org.destinationsol.assets.emitters.Emitter;
+import org.destinationsol.assets.fonts.Font;
+import org.destinationsol.assets.json.Json;
+import org.destinationsol.assets.textures.DSTexture;
+import org.terasology.assets.ResourceUrn;
+import org.terasology.module.ModuleEnvironment;
+
+import java.util.Optional;
+
+public abstract class Assets {
```

Comment:
Sir yes sir, on it! :D

(Gonna make a mega commit with all my classes doc-ed.. Ah, this will be painful)